spot-contracts/contracts/FeePolicy.sol

```diff
@@ -78,13 +80,9 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
     ///      Adds a safety buffer to ensure that rollovers are better sustained.
     uint256 public targetSubscriptionRatio;
 
-    /// @notice The lower bound of deviation ratio, below which some operations (which decrease the dr) are disabled.
-    uint256 public deviationRatioBoundLower;
-
-    /// @notice The upper bound of deviation ratio, above which some operations (which increase the dr) are disabled.
-    uint256 public deviationRatioBoundUpper;
-
-    //-----------------------------------------------------------------------------
+    /// @notice The lower and upper deviation ratio bounds outside which
+    ///         some operations (like flash swaps) which move dr away from 1.0 are disabled.
+    Range public drHardBounds;
```

Comment:
We'll be deploying a new FeePolicy given significant upcoming changes, so we can safely ignore any changes to the storage layout

Reply:
Combining these two properties into a struct is cleaner from a programming perspective, but I feel like it will make updating these values in the setter harder.

For example, when constructing a governance payload. Still worth it?

Reply:
Yea, haven't really any issue with complex struct payloads.